classes/functions.php

```diff
@@ -218,7 +218,27 @@ public function seriesList() {
 			die ( "Connection failed:" );
 		}
 
-		$sql = "SELECT * FROM series";
+		$sql = "SELECT * FROM series ORDER BY series_name ASC";
 		$this->series_list_result = $this->db_connection->query ( $sql );
 	}
+
+	public function seriesFind($series_name) {
+		$this->db_connection = new mysqli ( DB_HOST, DB_USER, DB_PASS, DB_NAME );
+		if ($this->db_connection->connect_errno) {
+			die ( "Connection failed:" );
+		}
+
+		$sql = "SELECT * FROM series where series_name = '$series_name'";
+		$this->series = $this->db_connection->query ( $sql );
+	}
+
+	public function seriesLatestCover($series_id) {
+		$this->db_connection = new mysqli ( DB_HOST, DB_USER, DB_PASS, DB_NAME );
+		if ($this->db_connection->connect_errno) {
+			die ( "Connection failed:" );
+		}
+		$sql = "SELECT cover_image FROM comics WHERE series_id = $series_id ORDER BY issue_number DESC LIMIT 1";
+		$result = $this->db_connection->query ( $sql );
+		$this->series_latest_cover = implode(mysqli_fetch_row($result));
+	}
 }
```

Comment:
This new function `seriesLatestCover` is what finds the most recent issue of the series and uses its downloaded cover as the thumbnail for the series. If no issues are added, or no images exist, it will display a default "nocover.jpg" image located in /assets/